spring-cloud-commons/src/main/java/org/springframework/cloud/commons/util/IdUtils.java

```diff
@@ -29,6 +29,7 @@
 	// @checkstyle:off
 	public static final String DEFAULT_SERVICE_ID_STRING = "${vcap.application.name:${spring.application.name:application}}:${vcap.application.instance_index:${spring.application.index:${local.server.port:${server.port:0}}}}:${vcap.application.instance_id:${cachedrandom.${vcap.application.name:${spring.application.name:application}}.value}}";
```

Comment:
Mark this as `@Deprecated` so we can remove it in the future

Reply:
Resolved by a51f4b2.

Reply:
Superseded by 91aaf03.